src/Resources/ResourceManager/Components/Constants.cs

```diff
@@ -77,17 +77,17 @@ public static class Constants
         /// <summary>
         /// The default policy definition API version.
         /// </summary>
-        public static readonly string PolicyDefinitionApiVersion = "2018-05-01";
+        public static readonly string PolicyDefinitionApiVersion = "2019-01-01";
```

Comment:
How does this not impact any tests?  Are there ary tests for Policy definition, and if so, why does this not invalidate those recordings?

Reply:
I think the tests didn't break as the schema of the API didn't change with new 2019-01-01 version. We have exposed new api version to enable data plane policies and to protect the existing clients from querying data plane policies. I am updating the recordings.